### ISSUE TITLE
Support for keyboard shortcut components

### ISSUE THREAD

**Issue by zsherman** [timberio/gitdocs#27]
Could do this with a custom component, but would be great to be able to render shortcuts like this https://cl.ly/233J0J1F0L3D.

**Comment by etimberg** [comment]
Good idea! I seem to recall a way to do this in Unicode ... trying to find something on google now

**Comment by etimberg** [comment]
Couldn't find much ... I think a custom component is the way to go. Makes it easier to control styling cross platform.
